### PR TITLE
feat(web-app): add basic code mdx support

### LINKED ISSUE #595: Render markdown code blocks and inline code
https://gatsby-theme-carbon.vercel.app/components/markdown#code-blocks

Render inline and block code using out of the box Carbon components. 

Additional functionality tracked here https://github.com/carbon-design-system/carbon-platform/issues/363

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #595 

#### Testing / reviewing

http://localhost:3000/mdx/code

### CODE REVIEW

File: services/web-app/components/code/code.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import { CodeSnippet, Column, Grid, Theme } from '@carbon/react'
+
+import styles from './code.module.scss'
+
+const Code = ({ children }) => (
+  <Theme theme={'g100'}>
+    <Grid>
+      <Column sm={4} md={6} lg={8}>
+        <CodeSnippet type="multi" wrapText="true" feedback="Copied!" className={styles.code}>
```
Comment (francinelucca) [inline services/web-app/components/code/code.js:16]: React props passed with just their name have a value of true
```suggestion
        <CodeSnippet type="multi" wrapText feedback="Copied!" className={styles.code}>
```

image

**Review by francinelucca — CHANGES_REQUESTED** [review]
is the first line supposed to start a left-er compared to other lines? 
image

**Comment by alisonjoseph** [comment]
Good catch @francinelucca, fixed! (no need to re-review yet, will add storybook to this PR once that is merged in)

**Comment by alisonjoseph** [comment]
@francinelucca this doesn't need to wait for storybook, this is just markdown, not an actual component.

**Review by francinelucca — DISMISSED** [review]
🎉

**Review by andreancardona — DISMISSED** [review]
🎉  just one question / comment here

File: services/web-app/components/code/code.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      <Column sm={4} md={6} lg={8}>
+        <CodeSnippet type="multi" wrapText feedback="Copied!" className={styles.code}>
+          {children}
+        </CodeSnippet>
+      </Column>
+    </Grid>
+  </Theme>
+)
+
+export default Code
```
Comment (andreancardona) [inline services/web-app/components/code/code.js:24]: Do we want do define PropTypes for children here?

Comment (alisonjoseph) [reply services/web-app/components/code/code.js:30]: Added propTypes

**Review by jharvey10 — APPROVED** [review]

**Review by francinelucca — APPROVED** [review]

**Pull request merged**